litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -530,6 +530,14 @@ spec:
       labels:
         component: litmusportal-server
     spec:
+      initContainers:
+        - name: wait-for-mongodb
+          image: curlimages/curl:latest
+          command: ["/bin/sh", "-c"]
+          args:
+            [
+              "while [[ $(curl -sw '%{http_code}' http://mongo-service.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local:27017 -o /dev/null) -ne 200 ]]; do sleep 5; echo 'Waiting for the MongoDB to be ready...'; done; echo 'Connection with MongoDB established'",
```

Comment:
ns name should not be needed I guess, you can directly just use `mongo-service` like we do in the server